### PR TITLE
VZ-2394 system fluentd for application logging

### PR DESCRIPTION
**Pull request by shihchang**
# Description

* Change Weblogic and Coherence Fluentd sidecar output from elasticsearch to stdout so that the logs can be collected by system Fluentd daemonset which ships all the logs to the elasticsearch in admin cluster.
* Clean up helidon and ContainerizedWorkload fluentd sidecar.
* Change e2e tests to assert the new namespace indices.

Fixes VZ-2394

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [x] Added or updated unit tests for any new functions I added
- [x] Added or updated integration tests if appropriate
- [x] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: application-operator/controllers/loggingscope/fluentd.go [hunk 1]
```diff
@@ -29,21 +25,13 @@ const (
 	fluentdConfKey       = "fluentd.conf"
 	fluentdConfMountPath = "/fluentd/etc/fluentd.conf"
 	configMapName        = "fluentd-config"
+	confVolume           = "fluentd-config-volume"
```
Comment (guoyongzhang) [inline application-operator/controllers/loggingscope/fluentd.go:28]: Is this being used?

Comment (shihchang) [reply application-operator/controllers/loggingscope/fluentd.go:28]: yes, wls and coh are still using fluentd sidecar which needs config, but the sidecar no longer connect to es but output to stdout.

File: application-operator/controllers/loggingscope/fluentd.go [hunk 2]
```diff
@@ -29,21 +25,13 @@ const (
...
-
-	CAFileConfig = "\n  ca_file /fluentd/secret/ca-bundle"
 )
 
-// ElasticSearchIndex defines the common index pattern
-const ElasticSearchIndex = "#{ENV['NAMESPACE']}-#{ENV['APP_CONF_NAME']}-#{ENV['COMPONENT_NAME']}"
-
 const (
 	// DefaultElasticSearchURL defines the default Elasticsearch URL used if it is not specified in the logging scope
 	DefaultElasticSearchURL = "http://vmi-system-es-ingest.verrazzano-system.svc.cluster.local:9200"
```
Comment (guoyongzhang) [inline application-operator/controllers/loggingscope/fluentd.go:37]: where would this be used?

**Comment by guoyongzhang** [comment]
When a custom loggingScope is specified in an app with an external ES URL and secret, would logs go there?

File: application-operator/controllers/loggingscope/fluentd.go (hunk 2 above)
Comment (shihchang) [reply application-operator/controllers/loggingscope/fluentd.go]: not until we support custom output

**Review by tbeerbower — APPROVED** [review]
Looks good to me.

**Pull request merged**